src/twisted/internet/testing.py

```diff
@@ -577,15 +578,16 @@ def fireSystemEvent(self, eventType):
         raise NotImplementedError()
 
 
-    def addSystemEventTrigger(self, phase, eventType, callable, *args, **kw):
+    def addSystemEventTrigger(self, phase: str, eventType: str,
+                              f: Callable[..., Any], *args, **kw):
```

Comment:
I guess that for the testing code this is ok, but is better to have this in the separate PR.

Reply:
I took this out